apps/Wallet/src/store/wallet/safe-address-slice.ts

```diff
@@ -0,0 +1,11 @@
+import { StateCreator } from "zustand";
+
+export type SafeAddressSlice = {
+  safeAddress: string;
+  setSafeAddress: (address: string) => void;
+};
+
+export const createSafeAddressSlice: StateCreator<SafeAddressSlice, [], [], SafeAddressSlice> = (set) => ({
+  safeAddress: "0x1f29deFa8a6472f65e1c86cD8E449cd37c521A41",
```

Comment:
0x08449253561c4515CFba82308c190f9D290D68fB

Reply:
will be empty once dev is finished